### DISCUSSION TITLE
Eclipse

### DISCUSSION THREAD

**Discussion by PatrickNotStar23** [catppuccin/catppuccin#1867] [Port Requests]
Are you available to help in the creation of this port? 😄 (optional)
Maybe (busy with college)

Additional context 📷 (optional)
Custom Eclipse IDE themes can be made with the [devstyle](https://www.genuitec.com/products/devstyle/) plugin. See [Dracula](https://draculatheme.com/eclipse)'s implementation.

**Comment by zarchinard** [comment]
Hey, I'm available to help in the creation of this port. How do I proceed ?

**Reply by sgoudham** [reply]
Hey 👋,
For the repository setup, you'll want to follow the instructions in https://github.com/catppuccin/catppuccin/blob/main/docs/port-creation.md#creation.

Then, as the author mentions above, you'll want to figure out how the devstyle plugin works in Eclipse and how we can offer the Catppuccin themes through it.

From looking at Dracula's implementation, it seems like it's driven by `.xml` files. I would highly recommend the use of [catppuccin/whiskers](https://github.com/catppuccin/whiskers), our new port creation helper tool, to automatically generate the themes based on a template file.

Feel free to join our [discord](https://discord.com/servers/catppuccin-907385605422448742) if you're looking to collaborate more in real-time!

**Comment by sgoudham** [comment]
Closing as any further discussion should take place in https://github.com/catppuccin/catppuccin/issues/2611
